charts/ChartView.tsx

```diff
@@ -69,6 +70,7 @@ export class ChartView extends React.Component<ChartViewProps> {
 
         render()
         window.addEventListener("resize", throttle(render))
+        urlBinding.bindUrlToWindow(chart.url)
```

Comment:
There should be an `isSinglePage` check here?

Reply:
Good catch… but this is in the bootstrap call, wouldn't `isSinglePage` always be true in that case?

Reply:
`bootstrap` is also used for the (iframeless) embeds: https://github.com/owid/owid-grapher/blob/d0b59de2fb930aec763c7aa24bee07db9845fa20/site/client/Grapher.ts#L81

Reply:
I moved this to a different function that is only called from the top level, and I got rid of isSinglePage completely